sdk/core/azure-core-amqp/inc/azure/core/amqp/connection.hpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-Licence-Identifier: MIT
+
+#pragma once
+
+#include "common/async_operation_queue.hpp"
+#include "connection_string_credential.hpp"
+#include "models/amqp_value.hpp"
+#include "session.hpp"
+#include <azure/core/credentials/credentials.hpp>
+#include <chrono>
+#include <memory>
+#include <string>
+
+namespace Azure { namespace Core { namespace Amqp {
+
+  namespace _detail {
+    class ConnectionImpl;
+  }
+  namespace _internal {
+    class Session;
+
+    struct ConnectionOptions
+    {
+      //    std::chrono::seconds IdleTimeout{std::chrono::seconds(1)};
```

Comment:
Remove commented out code.

Reply:
This will come back in my next PR, no point in removing it now.